### PR TITLE
Fix for CLI issues and improvment

### PR DESCRIPTION
**Pull request by PravinRanjan10**
**What this PR does / why we need it**:
1. This PR fixes the CLI related issues(which we found during QIP)
2. In CLI, Servicebility issues founded for profile and volume. There were no examples and proper help messages.
3. In CLI, few validations issues founded for volume and profile.
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes # #1208 #1206 #1218 #1202 #1147

**Special notes for your reviewer**:

1.Fix for issue: #1208 #1206 #1218 #1202 #1147

**Release note**:

```release-note
```

### CODE REVIEW

File: osdsctl/cli/volume.go [hunk 1]
```diff
@@ -109,13 +110,14 @@ func init() {
 	volumeListCommand.Flags().StringVarP(&volProfileId, "profileId", "", "", "list volume by profile id")
 	volumeListCommand.Flags().StringVarP(&volGroupId, "groupId", "", "", "list volume by volume group id")
 
-	volumeCommand.PersistentFlags().StringVarP(&profileId, "profile", "p", "", "the id of profile configured by admin")
+	volumeCommand.PersistentFlags().StringVarP(&profileId, "profile", "", "", "the id of profile configured by admin")
```
Comment (joseph-v) [inline osdsctl/cli/volume.go]: Please see if we can maintain previous option -p, and use different option for "--pool/-p"

**Comment by kumarashit** [comment]
Please put proper comments for :
What this PR does / why we need it:

Which issue this PR fixes (optional, in fixes #<issue number>(, fixes #<issue_number>, ...) format, will close that issue when PR gets merged): fixes #

Special notes for your reviewer:

Fix for CLI issue:1208,1218 and few more CLI improvment

Release note:

File: osdsctl/cli/volume.go (hunk 1 above)
Comment (PravinRanjan10) [reply osdsctl/cli/volume.go]: done

**Comment by joseph-v** [comment]
LGTM

**Pull request merged**